fastlane/report.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, the .gitignore should prevent this. Do you maybe not have the latest from master?